core/common/modules/connect/module.php

```diff
@@ -74,6 +74,10 @@ public function __construct() {
 			// Note: The priority 11 is for allowing plugins to add their register callback on elementor init.
 			add_action( 'elementor/init', [ $this, 'init' ], 11 );
 		}
+
+		add_filter( 'elementor/tracker/send_tracking_data_params', function ( $params ) {
+			return $this->add_tracking_data( $params );
+		} );
```

Comment:
Instead of using additional function, you can make `add_tracking_data` public and use this syntax:

```suggestion
		add_filter( 'elementor/tracker/send_tracking_data_params', [ $this, 'add_tracking_data' ] );
```

Reply:
According to our guidelines, this how we should do it unless its need to be protected or public.